Makefile

```diff
@@ -1,5 +1,5 @@
-# For musl, use: armv7-unknown-linux-musleabihf
-TARGET ?= armv7-unknown-linux-gnueabihf
+# For non-musl, use: armv7-unknown-linux-gnueabihf
+TARGET ?= armv7-unknown-linux-musleabihf
```

Comment:
I should probably revert this as this value is used in this Makefile for `cargo` calls, not just `cross` calls...

Reply:
Sounds solid. I'm fine with either way.